go-controller/pkg/testing/kube.go

```diff
@@ -0,0 +1,84 @@
+package testing
+
+import (
+	discovery "k8s.io/api/discovery/v1"
+	utilpointer "k8s.io/utils/pointer"
+)
+
+// USED ONLY FOR TESTING
+
+var NodeA string = "node-a"
+var NodeB string = "node-b"
```

Comment:
Can't these be passed as parameters to the utility methods below instead of being defined as pseudo constants?